src/test_double.py

```diff
@@ -58,33 +57,29 @@ def test_dnode_init_value(init_value):
     assert test_dnode.value == init_value
 
 
```

Comment:
I changed some of the test per Cris's comments on our linked list submission.  We should test to see if all the different types are getting in and out of the nodes correctly, but we would not need to check to see if all the different types are having the pointers initialize correctly.  Those functionality are independent of each other and if it does it for one, it would do it for all.  Thats why we see less test now when we tox it.